OnbordingTemplates/DeployDcr/PerfAndMapDcr/DeployDcrTemplate.json

```diff
@@ -0,0 +1,108 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "WorkspaceResourceId": {
+            "type": "String",
+            "metadata": {
+                "description": "Workspace Resource ID."
+            }
+        },
+        "WorkspaceLocation": {
+            "type": "String",
+            "metadata": {
+                "description": "Workspace Location."
+            }
+        },
+        "userGivenDcrName": {
+            "type": "String",
+            "metadata": {
+              "displayName": "Name of the Data Collection Rule(DCR)",
+              "description": "This is the name of the AMA-VMI Data Collection Rule(DCR)"
+            },
+            "defaultValue": "ama-vmi-default-dcr"
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2017-05-10",
+            "name": "[concat('VMI-DCR-Deployment-',uniqueString(parameters('userGivenDcrName')))]",
+            "properties": {
+                "mode": "Incremental",
+                "template": {
+                    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+                    "contentVersion": "1.0.0.0",
+                    "parameters": {},
+                    "variables": {},
+                    "resources": [
+                        {
+                            "type": "Microsoft.Insights/dataCollectionRules",
+                            "apiVersion": "2021-04-01",

```

Comment:
Is this the latest api version? Do we need to update the api?

Reply:
Wil review that, than you for your comments.